### PR TITLE
Extend scripts and fix metrics

### PR DESCRIPTION
**Pull request by miiu96**
- Add commands in the `Makefile` for the monitoring tools ( the new commands will start a server of Prometheus and Grafana)
- Fix  the error metrics

### CODE REVIEW

**Review by bogdan-rosianu — DISMISSED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

File: scripts/observers/Makefile [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 help:
 	@echo "Available targets:"
-	@echo "  config  - Configure the local testnet"
-	@echo "  start   - Start the local testnet"
-	@echo "  stop    - Stop the local testnet"
-	@echo "  clean   - Clean up the files of the local testnet"
-	@echo "  cluster - Start an Elasticsearch cluster on port 9200"
+	@echo "  config        - Configure the local testnet"
+	@echo "  start         - Start the local testnet"
+	@echo "  stop          - Stop the local testnet"
```
Comment (ssd04) [inline scripts/observers/Makefile:6]: suggestion to add in a next PR: add `install` command to handler prerequisites, like python install requirements, `screen`, ... , in order to be able to run the entire setup more easily if trying for the first time

Comment (ssd04) [reply scripts/observers/Makefile:6]: maybe also commands to handle the entire setup like:
`full-start: start cluster monitor-start`
`full-stop: stop cluster-stop monitor-stop`

**Review by ssd04 — APPROVED** [review]

File: scripts/observers/Makefile (hunk 1 above)
Comment (miiu96) [reply scripts/observers/Makefile:6]: Will make a task for this

Comment (miiu96) [reply scripts/observers/Makefile:6]: [MX-14453]

**Pull request merged**